docs/theme.md

```diff
@@ -0,0 +1,37 @@
+`zcli theme`
+===========
+
+zcli theme commands helps with managing Zendesk Help Center theming workflow.
+
+* [`zcli theme:preview [THEMEPATH]`](#zcli-themepreview-themepath)
+
+## Configuration
+
+NOTE: You can set your theme config/settings in `zcli.theme.config.json` at the root of your theme directory ie
+
+```
+{
+  "subdomain": "mysubdomain",
+  "username": "me@email.com",
```

Comment:
I agree but `zat` already used `username` 😕 
I think it makes sense to align to have a seamless migration from `zat` to `zcli`. It would basically be renaming the `.zat` file to `zcli.theme.config.json`.
Of course we can also change it and document it so lmk what you think!

Reply:
@anpa if it's ok with you I'm thinking of leaving this one as `username` since it seems to also be the default naming for HTTP Basic auth, e.g. `Authorization` header.
Do lmk if you have a string opinion about it though 🙏🏼